net/softethervpn/Makefile

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2008-2016 OpenWrt.org
```

Comment:
Far from looking for any contract with Openwrt, I am just one contributor of the Openwrt packages project. All previous packages I added (since 2014) had ALWAYS that copyright and moreover it seems to be present in EVERY makefile of the Github repository (I do not think that every contributor here has a contract with OpenWRT). This makefile has been done by my person, taking the work done by el1n and improving it as stated in the commit message; I have also inserted a reference to el1n in the same Makefile under the copyright (# Based on the initial porting done by el1n) and before doing this package I have asked obviously to el1n the permission to borrow his work after I understood he was not interested in merging his job into OpenWRT. I am not a "thief" if this is what you are meaning and anyone can verify this by comparing my makefile and the one done by el1n (https://github.com/el1n/OpenWRT-package-softether/blob/master/softethervpn/Makefile), there a lot of changes. I wanted to bring just another useful package to the repository since nobody was willing to do this for a long time. I do not understand why you are coming out with this absurd rationale, but if you do not want this package to be merged for any reasons or have a bad day fine, but please simply tell.